tljh/configurer.py

```diff
@@ -191,6 +203,43 @@ def update_traefik_api(c, config):
     c.TraefikTomlProxy.traefik_api_password = config['traefik_api']['password']
 
 
+def set_cull_idle_service(c, config):
+    """
+    Set Idle Culler service
+    """
+    cull_cmd = [
+       sys.executable, '/srv/src/tljh/cull_idle_servers.py'
+    ]
+    if config['services']['cull']['timeout']:
+        cull_cmd.append('--timeout=%s' % config['services']['cull']['timeout'])
```

Comment:
I always try to avoid string interpolation in creatin commands, to protect against [command injection](https://www.owasp.org/index.php/Command_Injection). I *think* in this case we are safe, but would be better to do it like:

```python
cull_cmd += ['--timeout', str(cull_config['timeout']]
```

Reply:
@yuvipanda, I noticed that [parse_command_line](https://www.tornadoweb.org/en/stable/_modules/tornado/options.html#OptionParser.parse_command_line) used by *cull_idle_servers.py* requires the options to be passed in like ``--option=value`` and complains about having them like ``--option value``.
I could enforce the type of the option value in *cull_idle_servers.py* like this:
```
define('timeout', type=int, default=600, help="The idle timeout (in seconds)")
```
and then add it to the command like this:
```
cull_cmd += ['--timeout=%d' % cull_config['timeout']]
```
Do you believe this approach will reduce enough the risks of a command injection ?

Reply:
@yuvipanda, I've addressed your feedback and defined the *cull_cmd* as described above. Please let me know what you think :D
About the documentation, where do you believe it is better to add it? I'm thinking about *Administration and security* or *Topic Guides* :thinking:

Reply:
@GeorgianaElena great! I'll take a look shortly. I think 'Topic' is a great place to start here.